src/appleseed/renderer/api/postprocessing.h

```diff
@@ -37,3 +37,6 @@
 #include "renderer/modeling/postprocessingstage/postprocessingstagetraits.h"
 #include "renderer/modeling/postprocessingstage/renderstamppostprocessingstage.h"
 #include "renderer/modeling/postprocessingstage/vignettepostprocessingstage.h"
+#include "renderer/modeling/postprocessingstage/effect/ipostprocessingeffect.h"
```

Comment:
I don't think this needs to be exposed to the API. I think we expose only stages and that should be enough to apply post process.